### DISCUSSION TITLE
Cannot use with .net6 through NUGET.

### DISCUSSION THREAD

**Discussion by lanpst** [ArxOne/MrAdvice#185] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6269204/167782266-7dc999a8-1f66-4d25-a48d-ce5b01e8c7a8.png)
I get MrAdvice library through NUGET. but there is a error when i debug the code.
I find in your main page that it is only support .net 5. So is there a plan to support .net 6? 
This is really a good library and we expect to use it in .net6.

**Comment by picrap** [comment]
It supports .net6 and we use it with .net6, so I am confused.
Do you have details on the error? Does the `MrAdvice.Weaver.dll` exist at the given location?

**Reply by lanpst** [reply]
> It supports .net6 and we use it with .net6, so I am confused. Do you have details on the error? Does the `MrAdvice.Weaver.dll` exist at the given location?

i can find the dll file in this path:
C:\Users\Lanny Ling\.nuget\packages\mradvice\2.9.7\lib\net6.0\MrAdvice.dll
because the VS cannot debug which means the file will not copy to the debug folder.

and i refer this dll file directly without NUGET and there is no error when DEBUG. 
but the interception cannot work which means the breakpoint cannot stop into the method.
![image](https://user-images.githubusercontent.com/6269204/167831370-c54bcf87-8582-441e-8c26-210d098a17eb.png)

**Comment by lanpst** [comment, marked as answer]
I find the solution that maybe my net version is not latest.
in .net 6.0.5.the problem is solved.

**Reply by picrap** [reply]
Good news, then. Thank you.

**Comment by XuLinFei** [comment]
I also encountered the same problem, but I upgraded to the latest SDK of net6 and it didn't work. I introduced it through nuget
